starter-code/routes/authController.js

```diff
@@ -0,0 +1,113 @@
+const express        = require("express");
+const authController = express.Router();
+
+// User model
+const User           = require("../models/user");
+
+// Bcrypt to encrypt passwords
+const bcrypt         = require("bcrypt");
+const bcryptSalt     = 10;
+
```

Comment:
missing a `authController.use(...)` to protect logged in users to view and/or submit login/signup

just like you did in `welcomeController.js` but with the reversed logic:
- is there a user in the session? redirect
- else, `next()`